docs/advanced_syntax/builtins.md

```diff
@@ -0,0 +1,50 @@
+# Builtins
+
+Builtins are native methods (built in the compiler itself) reserved keywords in Amber.  
```

Comment:
Reading this line without the parenthesis part, the `... are native methods reserved keywords ...` doesn't make sense. It doesn't with the parenthesis part, either. The problem is that it is defining multiple things ("methods", "reserved keywords") as some one thing.